### PR TITLE
Edits to PGD PR5477

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/pgd/5/sequences.mdx [hunk 1]
```diff
@@ -18,8 +18,8 @@ produce values that are unique only on the local node. This is important because
 unique ids generated by such sequences cause conflict and data loss by
 means of discarded `INSERT` actions in multi-master replication.
 
-!!! Note Permissions Required
-PGD global sequences are on by default. This means that for any user to use sequences, 
+!!! Note Permissions required
+PGD global sequences are on by default. This means that, to use sequences,
```
Comment (djw-m) [inline product_docs/docs/pgd/5/sequences.mdx:22]: This note is making the point that for ANY user to use sequences they must have the appropriate role. Sequences are embedded and other users using the table are required to have the same role privilege - hence for any user rather than the user.

**Review by djw-m — CHANGES_REQUESTED** [review]

File: product_docs/docs/pgd/5/sequences.mdx [hunk 2]
```diff
@@ -18,9 +18,9 @@ produce values that are unique only on the local node. This is important because
 unique ids generated by such sequences cause conflict and data loss by
 means of discarded `INSERT` actions in multi-master replication.
 
-!!! Note Permissions Required
-PGD global sequences are on by default. This means that for any user to use sequences, 
-the user must have at least the [bdr_application](security/pgd-predefined-roles/#bdr_application) 
+!!! Note Permissions required
+PGD global sequences are on by default. This means that any user who wants to use sequences, 
+must have at least the [bdr_application](security/pgd-predefined-roles/#bdr_application)
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/sequences.mdx:23]: @djw-m Does this work? The original was hard to parse because of the two uses of the word "user" in the sentence.

File: product_docs/docs/pgd/5/consistency/column-level-conflicts.mdx [hunk 3]
```diff
@@ -36,7 +36,7 @@ Column-level conflict resolution requires the table to have `REPLICA IDENTITY FU
 ## Enabling and disabling column-level conflict resolution
 
 !!! Note Permissions required
-Column-level conflict detection uses the `column_timestamps` type. This requires the user to have at least the [bdr_application](../security/pgd-predefined-roles/#bdr_application) role assigned to them.
+Column-level conflict detection uses the `column_timestamps` type. This type requires you to have at least the [bdr_application](../security/pgd-predefined-roles/#bdr_application) role assigned.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/consistency/column-level-conflicts.mdx:39]: @djw-m This negates our previous exchange...Not sure why you used second person here? If you mean a specific user (i.e., someone who's doing something specific) can you be more explicit?

File: product_docs/docs/pgd/5/consistency/crdt.mdx [hunk 4]
```diff
@@ -7,8 +7,8 @@ redirects:
 
 Conflict-free replicated data types (CRDT) support merging values from concurrently modified rows instead of discarding one of the rows as traditional resolution does.
 
-!!! Note Permissions Required
-PGD CRDT require usage access to CRDT types, therefore a user must have at least the [bdr_application](../security/pgd-predefined-roles/#bdr_application) role assigned to them.
+!!! Note Permissions required
+PGD CRDT requires usage access to CRDT types. Therefore a user must have at least the [bdr_application](../security/pgd-predefined-roles/#bdr_application) role assigned to them.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/consistency/crdt.mdx:11]: I have the same query here as to why this is in third person.

File: product_docs/docs/pgd/5/sequences.mdx [hunk 5]
```diff
@@ -19,8 +19,8 @@ produce values that are unique only on the local node. This is important because
 unique ids generated by such sequences cause conflict and data loss by
 means of discarded `INSERT` actions in multi-master replication.
 
-!!! Note Permissions Required
-PGD global sequences are on by default. This means that for any user to use sequences, 
+!!! Note Permissions required
+PGD global sequences are on by default. This means that, to use sequences,
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/sequences.mdx:23]:
```suggestion
This means that any user who wants to use sequences
```

File: product_docs/docs/pgd/5/sequences.mdx [hunk 6]
```diff
@@ -19,8 +19,8 @@ produce values that are unique only on the local node. This is important because
 unique ids generated by such sequences cause conflict and data loss by
 means of discarded `INSERT` actions in multi-master replication.
 
-!!! Note Permissions Required
-PGD global sequences are on by default. This means that for any user to use sequences, 
+!!! Note Permissions required
+PGD global sequences are on by default. This means that, to use sequences, 
 the user must have at least the [bdr_application](security/pgd-predefined-roles/#bdr_application)
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/sequences.mdx:24]:
```suggestion
must have at least the [bdr_application](security/pgd-predefined-roles/#bdr_application) 
```

**Comment by ebgitelman** [comment]
@djw-m Hoping we can wrap up these changes...

File: product_docs/docs/pgd/5/consistency/crdt.mdx (hunk 4 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/consistency/crdt.mdx:11]:
```suggestion
PGD CRDT requires usage access to CRDT types. Therefore any user needint to access CRDT types must have at least the [bdr_application](../security/pgd-predefined-roles/#bdr_application) role assigned to them.
```

**Review by djw-m — APPROVED** [review]
LGTM
